test_utils/fake_catalog_api.py

```diff
@@ -0,0 +1,263 @@
+        "start": "2015-01-01T00:00:00Z",
+        "end": None,
+        "video": None,
+        "seats": [],
+        "programs": [
+            {
+                "uuid": "40782a06-1c37-4779-86aa-0a081f014d4d",
+                "title": "Program1",
+                "type": "Prof only",
+                "marketing_slug": "program1",
+                "marketing_url": "prof only/program1"
+            }
+        ]
+    },
+    {
+        "key": "course-v1:Organization+ENT-1+T1",
+        "uuid": "a2128a84-6e20-4fce-958e-80d9461ef835",
+        "title": "Enterprise Tests",
+        "course": "course-v1:Organization+ENT-1+T1",
+        "start": None,
+        "end": None,
+        "seats": [
+            {
+                "type": "professional",
+                "price": "0.00",
+                "currency": "AZN",
+                "upgrade_deadline": None,
+                "credit_provider": "",
+                "credit_hours": None
+            },
+            {
+                "type": "audit",
+                "price": "0.00",
+                "currency": "AED",
+                "upgrade_deadline": None,
+                "credit_provider": "",
+                "credit_hours": None
+            }
+        ],
+        "programs": [
+            {
+                "uuid": "40782a06-1c37-4779-86aa-0a081f014d4d",
+                "title": "Program1",
+                "type": "Prof only",
+                "marketing_slug": "program1",
+                "marketing_url": "prof only/program1"
+            }
+        ]
+    },
+    {
+        "key": "course-v1:Organization+VD1+VD1",
+        "uuid": "5b949fc1-aa05-42b0-8c9f-8e6114848ae9",
+        "title": "VectorDraw",
+        "course": "Organization+VD1",
+        "start": "2030-01-01T00:00:00Z",
+        "end": None,
+        "seats": [
+            {
+                "type": "professional",
+                "price": "12.00",
+                "currency": "BOB",
+                "upgrade_deadline": None,
+                "credit_provider": "",
+                "credit_hours": None
+            }
+        ],
+        "programs": [
+            {
+                "uuid": "40782a06-1c37-4779-86aa-0a081f014d4d",
+                "title": "Program1",
+                "type": "Prof only",
+                "marketing_slug": "program1",
+                "marketing_url": "prof only/program1"
+            }
+        ]
+    }
+]
+
+FAKE_PROGRAM_RESPONSES = {
+    FAKE_PROGRAM_RESPONSE1["uuid"]: FAKE_PROGRAM_RESPONSE1,
+    FAKE_PROGRAM_RESPONSE2["uuid"]: FAKE_PROGRAM_RESPONSE2,
+}
+
+
+def get_program_by_uuid(program_uuid):
+    """
+    Fake implementation returning program by UUID.
+    """
+    return FAKE_PROGRAM_RESPONSES.get(program_uuid)
+
+
+def get_program_by_title(program_title):
+    """
+    Fake implementation returning program by title.
+    """
+    try:
+        return next(response for response in FAKE_PROGRAM_RESPONSES.values() if response["title"] == program_title)
```

Comment:
@haikuginger it is a good point, but it might make more sense to keep it as is, since it is a fake method - it is supposed to be simpler than actual implementation. Also, it won't have such problem (at least for now), since the only two programs avaiable to it have different titles.